infrastructure/variables.tf

```diff
@@ -58,11 +58,16 @@ variable "jenkins_AAD_objectId" {
   description                 = "(Required) The Azure AD object ID of a user, service principal or security group in the Azure Active Directory tenant for the vault. The object ID must be unique for the list of access policies."
 }
 
-variable "definition_cache_ttl_sec" {
+variable "definition_cache_max_idle_sec" {
   type = "string"
   default = "259200"
 }
 
+variable "definition_latest_version_ttl_sec" {
+  type = "string"
+  default = "30"
```

Comment:
let's use 1 as default, to be as close as possible to the current behaviour by default (no caching)

Reply:
Done so.